core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1515,6 +1520,18 @@ public void heartbeat() {
           }
         }
       }
+      for (MasterWorkerInfo worker : mLostWorkers) {
+        try (LockResource r = worker.lockWorkerMeta(
+                EnumSet.of(WorkerMetaLockSection.BLOCKS), false)) {
+          final long lastUpdate = mClock.millis() - worker.getLastUpdatedTimeMs();
+          if ((lastUpdate - masterWorkerTimeoutMs) > masterWorkerDeleteTimeoutMs) {
+            LOG.error("The worker {}({}) timed out after {}ms without a heartbeat! "
+                            + "it will be delete from the master", worker.getId(),
+                    worker.getWorkerAddress(), lastUpdate);
```

Comment:
I'm not too sure, but the best indentation should look like this? Pls double check :)
```suggestion
            LOG.error("The worker {}({}) timed out after {}ms without a heartbeat! "
                + "Master will forget about this worker.", worker.getId(),
                worker.getWorkerAddress(), lastUpdate);
```